query/README.md

```diff
@@ -0,0 +1,146 @@
+Keen Query Clients
+===================
+
+[![Build Status](https://travis-ci.org/keenlabs/KeenClient-Java.png?branch=master)](https://travis-ci.org/keenlabs/KeenClient-Java)
+
+The query capabilities within the Java Keen client enable you to send POST queries and receive the results of the queries in a JSON object. For query types, refer to API technical reference: https://keen.io/docs/api/reference/.
+
+## Usage
+### Building a Keen Query Client
+You can build a KeenQueryClient by just providing a KeenProject.
+```java
+KeenProject queryProject = new KeenProject("<project id>", "<write key>", "<read key>");
+KeenQueryClient queryClient = new TestKeenQueryClientBuilder(queryProject).build();
```

Comment:
Question: Did you intend to have `TestKeenQueryClientBuilder` in the README? I would have expected test code to be hidden from consumers of the SDK.